.eslintrc

```diff
@@ -105,6 +105,7 @@
     "jsx-a11y/label-has-for": 0,
     "linebreak-style": 0,
     "react/require-default-props": 0,
-    "react/sort-comp": 0
+    "react/sort-comp": 0,
+    "no-unused-vars": ["error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false, argsIgnorePattern: "event" }]
```

Comment:
There are some function [here]() that use the variable `event` so that we can pick user data. When we try and commit, we get `unused variable event` error. So we ignore the error specifically for the `event` variable.